content/en/docs/setup/install/multicluster/gateways/index.md

```diff
@@ -264,10 +264,13 @@ running in a second cluster. Before you begin:
     {{< /tip >}}
 
     If the global services have actual VIPs, you can use those, but otherwise we suggest
-    using IPs from the multicast range `224.0.0.0/4` that are not already allocated.
-    These IPs are not loopback addresses and are non-routable outside of a pod.
+    using IPs from the class E addresses range `240.0.0.0/4` that are preserved.
```

Comment:
What does "are preserved" mean? How can one know which ones are preserved?

Reply:
`Class E networks are defined by having the first four network address bits as 1. That encompasses addresses from 240.0.0.0 to 255.255.255.255. While this class is reserved, its usage was never defined. As a result, most network implementations discard these addresses as illegal or undefined. The exception is 255.255.255.255, which is used as a broadcast address.`